packages/avatars/src/styled/utility.ts

```diff
@@ -16,40 +16,35 @@ const [xxs, xs, s, m, l] = SIZE;
 export const TRANSITION_DURATION = 0.25;
 
 export interface IStyledStatusIndicatorProps extends ThemeProps<DefaultTheme> {
-  readonly size?: IAvatarProps['size'];
-  readonly type?: IAvatarProps['status'] | 'active';
+  readonly $size?: IAvatarProps['size'];
+  readonly $type?: IAvatarProps['status'] | 'active';
 }
 
 export function getStatusColor(
-  type?: IStyledStatusIndicatorProps['type'],
-  theme?: IStyledStatusIndicatorProps['theme']
+  theme: IStyledStatusIndicatorProps['theme'],
+  type?: IStyledStatusIndicatorProps['$type']
 ): string {
-  switch (type) {
-    case 'active':
-      return getColorV8('crimson', 400, theme)!;
-    case 'available':
-      return getColorV8('mint', 400, theme)!;
-    case 'away':
-      return getColorV8('orange', 400, theme)!;
-    case 'transfers':
-      return getColorV8('azure', 400, theme)!;
-    case 'offline':
-      return getColorV8('grey', 500, theme)!;
-    default:
-      return 'transparent';
-  }
+  return (
+    {
+      active: getColor({ hue: 'crimson', light: { shade: 700 }, dark: { shade: 600 }, theme }),
+      available: getColor({ hue: 'mint', light: { shade: 500 }, dark: { shade: 400 }, theme }),
+      away: getColor({ hue: 'orange', light: { shade: 500 }, dark: { shade: 400 }, theme }),
+      transfers: getColor({ hue: 'azure', light: { shade: 500 }, dark: { shade: 400 }, theme }),
+      offline: getColor({ hue: 'grey', light: { shade: 500 }, dark: { shade: 400 }, theme })
+    }[type as string] || 'transparent'
+  );
```

Comment:
```suggestion

  switch (type) {
    case 'active':
      return getColor({ hue: 'crimson', light: { shade: 700 }, dark: { shade: 600 }, theme });
    case 'available':
      return getColor({ hue: 'mint', light: { shade: 500 }, dark: { shade: 400 }, theme });
    case 'away':
      return getColor({ hue: 'mint', light: { shade: 500 }, dark: { shade: 400 }, theme });
    case 'transfers':
      return getColor({ hue: 'orange', light: { shade: 500 }, dark: { shade: 400 }, theme });
    case 'offline':
      return getColor({ hue: 'grey', light: { shade: 500 }, dark: { shade: 400 }, theme });
    default:
      return 'transparent';
  }
  
```

Reply:
Ah, that's what I had before.

I feel like there are two different trains of thought here. I ended up doing something like this: 403bf68d9c3fd10076501bf31c9501cc99c84de2